packages/frontend/src/pages/offers/pagination/Pagination.tsx

```diff
@@ -0,0 +1,51 @@
+import { AssetId } from '@explorer/types'
+import React from 'react'
+
+import { ServerPagination } from '../../common/pagination'
+import { OfferType } from '../ForcedTradeOffersIndexProps'
+import { AssetSelect } from './AssetSelect'
+import { TypeRadio } from './TypeRadio'
+
+interface PaginationProps {
+  page: number
+  perPage: number
+  total: number
+  assetId?: AssetId
+  type?: OfferType
+  assetIds?: AssetId[]
+}
+
+export function Pagination({
+  page,
+  perPage,
+  assetId,
+  assetIds,
+  type,
+  total,
+}: PaginationProps) {
+  const params = new URLSearchParams()
+  if (type) {
+    params.append('type', type)
+  }
+  if (assetId) {
+    params.append('assetId', assetId.toString())
+  }
+  const query = params.toString()
+  const baseUrl = `/forced/offers${query && `?${query}`}`
+
+  return (
+    <div className="relative pt-9">
+      <ServerPagination
```

Comment:
I am not sure I like the idea of ServerPagination wrapping other details. Pagination seems separate in a way, or at least in name. I get that this simplifies the js logic. Won't it work with two forms?